web/index.html

```diff
@@ -1,5 +1,5 @@
 <!doctype html>
-<html lang="en" data-theme="light">
```

Comment:
This was leftover from pico, which could use the browser's light/dark mode setting

Reply:
Yes, no worries. I think it should hopefully be reasonably obvious what does and doesn't matter. I started with your structure to avoid you needing to do much shifting.

Reply:
It was really easy to copy things over, so thank you, that approach worked well!